showcase/README.md

```diff
@@ -2,96 +2,149 @@
-|   |   |   crossdomain.xml             # cross domain policies
-|   |   |   favicon.ico                 # icon for bookmarks bar
-|   |   |   humans.txt                  # contains information about the website (http://humanstxt.org/)
-|   |   |   manifest.json               # application icons for Android mobile devices
-|   |   |   README.md                   #
-|   |   |   robots.txt                  # the robots exclusion protocol (http://www.robotstxt.org/)
-|   |   \   service-worker.js           # support for building Progressive Web Applications (PWA) with Service Workers
+|   |       browserconfig.xml           # application icons for Windows mobile devices
+|   |       crossdomain.xml             # cross domain policies
+|   |       favicon.ico                 # icon for bookmarks bar
+|   |       humans.txt                  # contains information about the website (http://humanstxt.org/)
+|   |       manifest.json               # application icons for Android mobile devices
+|   |       README.md                   #
+|   |       robots.txt                  # the robots exclusion protocol (http://www.robotstxt.org/)
+|   |       service-worker.js           # support for building Progressive Web Applications (PWA) with Service Workers
 |   |
 |   +---environments                    # configuration variables for each environment
-|   |   |                               #
-|   |   |   environment.e2e.prod.ts     # e2e tests configuration
-|   |   |   environment.hmr.ts          # development with HMR (Hot Module Replacement) configuration
-|   |   |   environment.prod.ts         # production configuration
-|   |   |   environment.ts              # development configuration
-|   |   \   model.ts                    # interface defining the variables
+|   |       environment.e2e.prod.ts     # production environment configuration for e2e tests
+|   |       environment.hmr.ts          # development with HMR (Hot Module Replacement) environment configuration
+|   |       environment.prod.ts         # production environment configuration
+|   |       environment.ts              # development environment configuration
 |   |
 |   +---styles                          # application-specific styles
-|   |   \
+|   |       ...
 |   |
 |   |   custom-typings.d.ts             # add your own type definitions that can't be found in the registry in this file
-|   |   hmr.ts                          # Hot Module Replacement logic
 |   |   index.html                      # the main HTML page that is served when someone visits this site
-|   |   main.browser.ts                 # Angular bootstrapping
-|   |   polyfills.browser.ts            # this file includes polyfills needed by Angular and should be loaded before the app
+|   |   main.browser.ts                 # main bundle entry point, Angular bootstrapping
+|   |   polyfills.browser.ts            # polyfills needed by old browsers if needed. Should be loaded before the app
```

Comment:
maybe "polyfills needed by old browsers, if required"? To avoid repetition?

Reply:
True, i'll fix it